dpctl/tests/test_sycl_kernel_submit.py

```diff
@@ -29,49 +29,44 @@
 from ._helper import has_gpu
 
 
-@unittest.skipUnless(has_gpu(), "No OpenCL GPU queues available")
-class Test1DKernelSubmit(unittest.TestCase):
-    def test_create_program_from_source(self):
-        oclSrc = "                                                             \
-        kernel void axpy(global int* a, global int* b, global int* c, int d) { \
-            size_t index = get_global_id(0);                                   \
-            c[index] = d*a[index] + b[index];                                  \
-        }"
-        q = dpctl.SyclQueue("opencl:gpu", property="enable_profiling")
-        prog = dpctl_prog.create_program_from_source(q, oclSrc)
-        axpyKernel = prog.get_sycl_kernel("axpy")
+@pytest.mark.skipif(not has_gpu(), reason="No OpenCL GPU queues available")
+def test_create_program_from_source():
+    oclSrc = "                                                             \
+    kernel void axpy(global int* a, global int* b, global int* c, int d) { \
+        size_t index = get_global_id(0);                                   \
+        c[index] = d*a[index] + b[index];                                  \
+    }"
+    q = dpctl.SyclQueue("opencl:gpu", property="enable_profiling")
```

Comment:
No need for the test to be skipped if GPU device is not available, but CPU is.

```suggestion
    q = dpctl.SyclQueue("opencl", property="enable_profiling")
```